iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/utils/CommandDetector.java

```diff
@@ -163,24 +157,16 @@ private void addCommandPredicate(CommandPredicate commandPredicate) {
       predicates.add(commandPredicate);
     }
 
-    private void addSingularPredicate(Predicate<String> predicate, CommandPredicate.Type type) {
+    private void addSingularPredicate(Predicate<String> predicate, Type type) {
       addCommandPredicate(new SingularPredicate(predicate, type));
     }
 
-    private void addOptionPredicate(SingularPredicate flag, SingularPredicate value) {
```

Comment:
These removed methods are dead code that was causing low test coverage

Reply:
@mstachniuk will be happy to see that the `CommandDetector` refactoring has started :smile: